examples/default/src/screens/LegacyModeScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { ActivityIndicator } from 'react-native';
+import Instabug from 'instabug-reactnative';
+import { ListTile } from '../components/ListTile';
+import { Screen } from '../components/Screen';
+import { showNotification } from '../utils/showNotification';
+
+export const LegacyModeScreen: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+
+  const addInstabugLog = (log: string): Promise<void> => {
+    return new Promise<void>((resolve) => {
+      Instabug.logDebug(log);
+      resolve();
+    });
+  };
+
+  const addUserEvent = (event: string): Promise<void> => {
+    return new Promise<void>((resolve) => {
+      Instabug.logUserEvent(event);
+      resolve();
+    });
+  };
+  const addTag = (tag: string): Promise<void> => {
+    return new Promise<void>((resolve) => {
+      Instabug.appendTags([tag]);
+      resolve();
+    });
+  };
+  const addUserAttributes = (key: string, value: string): Promise<void> => {
+    return new Promise<void>((resolve) => {
+      Instabug.setUserAttribute(key, value);
+      resolve();
+    });
+  };
+
+  const addMultipleInstabugLogs = async (numberOfLogs: number) => {
+    setLoading(true);
+    try {
+      const logPromises: Promise<void>[] = [];
+      for (let i = 0; i < numberOfLogs; i++) {
+        logPromises.push(addInstabugLog(`log ${i}`));
+      }
+      await Promise.all(logPromises);
```

Comment:
Since we will be removing the promises, I think we can simplify this (and similar methods below) to something like the following.

```suggestion
      for (let i = 0; i < numberOfLogs; i++) {
        addInstabugLog(`log ${i}`);
      }
```